CarbonFootprint/CarbonFootprint/Models/Email.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Solution.Models
+{
+    public class Email
+    {
+       public int numberOfInboxMail = 0;
```

Comment:
since count of inbox is getting returned . the variable name can be more specific like countOfInboxMail